### PR TITLE
[Task] Changing js class namespace, routing prefix and routing names

### PR DESCRIPTION
**Pull request by mattamon**
## Changes in this pull request  
Follow up to #13811 

Adding `bundle` as prefix for namespaces and routes for more consistency

### CODE REVIEW

File: bundles/GlossaryBundle/public/js/glossary.js [hunk 1]
```diff
@@ -11,8 +11,8 @@
  * @license    http://www.pimcore.org/license     GPLv3 and PCL
  */
 
-pimcore.registerNS("pimcore.settings.glossary");
-pimcore.settings.glossary = Class.create({
+pimcore.registerNS("pimcore.bundle.glossary");
+pimcore.bundle.glossary = Class.create({
```
Comment (brusch) [inline bundles/GlossaryBundle/public/js/glossary.js]:
```suggestion
pimcore.registerNS("pimcore.bundle.glossary.settings");
pimcore.bundle.glossary.settings = Class.create({
```

**Review by brusch — DISMISSED** [review]
We should put all JS files into the `pimcore.bundle.glossary` namespace, while `pimcore.bundle.glossary` shouldn't be used directly, I'd rather use `pimcore.bundle.glossary.init` or `pimcore.bundle.glossary.startup` for that.

**Comment by brusch** [comment]
@mattamon @dvesh3 For the glossary bundle we need to move the translations as well and they should be in the admin domain (e.g. `admin.en.json`).

File: bundles/GlossaryBundle/public/js/glossary.js (hunk 1 above)
Comment (mattamon) [reply bundles/GlossaryBundle/public/js/glossary.js]: I renamed the namespaces and also changed the glossary.js to settings.js to make it more consistent with the namespace. 
Did the same with the startup.js so the namespace is pimcore.bundle.glossary.startup to be consistent with the filenames.

Comment (brusch) [reply bundles/GlossaryBundle/public/js/glossary.js]: 👍 LGTM

**Review by dvesh3 — APPROVED** [review]

**Pull request merged**